client/js/lounge.js

```diff
@@ -72,6 +72,15 @@ $(function() {
 		return offset;
 	}
 
+	function isOpInChannel(channel) {
```

Comment:
Could we move this to a new file? Doesn’t seen to fit into lounge.js

Reply:
Move to utils.js maybe?